src/modularPlantManager/ModularPlantManager.spec.ts

```diff
@@ -258,52 +237,14 @@ describe('ModularPlantManager', () => {
 		});
 	});
 
-	/*	it('should reject loading PEAs with empty options', () => {
-			const modularPlantManager = new ModularPlantManager();
-			expect(() => modularPlantManager.loadPEAController({})).to.throw();
-			expect(() => modularPlantManager.loadPEAController({someattribute: 'abc'} as any)).to.throw();
-		});*/
-
-		/*it('should loadPEAControllerPEAController with single PEAController', () => {
-			const peasJson = JSON.parse(fs.readFileSync('assets/peas/pea_cif.json').toString());
-			const peaJson = peasJson.peas[0];
-			const modularPlantManager = new ModularPlantManager();
-			modularPlantManager.loadPEAController({pea: peaJson});
-			expect(() => modularPlantManager.loadPEAController({pea: peaJson})).to.throw('already in registered PEAs');
-		});
-
+	/*
 		it('should load with subMP options', () => {
 			const peasJson = JSON.parse(fs.readFileSync('assets/peas/pea_cif.json').toString());
 			const modularPlantManager = new ModularPlantManager();
 			modularPlantManager.loadPEAController({subMP: [peasJson]});
 			expect(() => modularPlantManager.loadPEAController({subMP: [peasJson]})).to.throw('already in registered PEAs');
 		});
 */
-		/*it('should load the Achema PEAs', async () => {
-			const modularPlantManager = new ModularPlantManager();
-			const peas = modularPlantManager.loadPEAController(
-				JSON.parse(fs.readFileSync('assets/peas/achema_demonstrator/peas_achema.json').toString()),
-				true);
-			expect(peas).to.have.lengthOf(3);
-
-			expect(modularPlantManager.loadPEAController).to.have.lengthOf(3);
-
-			const service = modularPlantManager.getService('Dose', 'Fill');
-			expect(service).to.be.instanceOf(Service);
-			expect(service.name).to.equal('Fill');
-			expect(() => modularPlantManager.getService('Dose', 'NoService')).to.throw();
-			expect(() => modularPlantManager.getService('NoPEA', 'NoService')).to.throw();
-
-			await expect(modularPlantManager.removePEAController('something')).to.be.rejectedWith('PEAController with id something not found');
-		});*/
-
-		it('should prevent removing a protected PEAController', async () => {
```

Comment:
Why did you remove this part?

Reply:
Hey, the achema json file doesn't exist anymore, so this test doesn't work. Should I keep it as a TODO?

Reply:
I have added it again